oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/ClaimAccessor.java

```diff
@@ -94,10 +94,18 @@ default String getClaimAsString(String claim) {
 	 * Returns the claim value as a {@code Boolean} or {@code null} if it does not exist.
 	 * @param claim the name of the claim
 	 * @return the claim value or {@code null} if it does not exist
+	 * @throws IllegalArgumentException if the claim value cannot be converted to a
+	 * {@code Boolean}
```

Comment:
For consistency, I think you should drop the `@throws`, as the `IllegalArgumentException` is self explanatory when it occurs.